queue_1.py

```diff
@@ -0,0 +1,54 @@
+from list import Node, List, ListIterator
+
+
+class Queue(List):
+    def __init__(self):
+        self.head = None
+        self.length = 0
+
+
+    def __getitem__(self, index):
+        self.check_index_borders(index)
+        if index != 0:
+            raise Exception(f"You cannot access to this element!")
+        return self.head.elem
+
+
+    def __setitem__(self, index, elem):
+        self.check_index_borders(index)
+        raise Exception(f"You cannot change value!")
+
+
+    def put(self, elem):
+        if self.head == None:
```

Comment:
This code is equal to push() method, although in its current form it even has a bug. The `tail` isn't set to the new node when it is an empty list